packages/storybook/migrations.json

```diff
@@ -75,6 +75,35 @@
     }
   },
   "packageJsonUpdates": {
+    "13.4.4": {
+      "version": "13.4.4-beta.1",
```

Comment:
oh, I think this should be 13.4.5-beta... 13.4.4 is already out there :)

Reply:
ha! of course!!